src/unittest/python/plugins/python/coverage_plugin_tests.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from pybuilder.core import Project
+from pybuilder.plugins.python.coverage_plugin import (
+    init_coverage_properties
+    )
+
+
+class CoveragePluginTests(TestCase):
+
+    def setUp(self):
+        self.project = Project("basedir")
+
+    def test_should_leave_user_specified_properties_when_initializing_plugin(self):
+
+        expected_properties = {
```

Comment:
These are the default values already. If we set them then we cannot know if the values we set were kept, or if the default values fell through.
We *absolutely* need to have non-default values here for the test to make sense.